zerver/webhooks/sonarr/view.py

```diff
@@ -44,115 +43,130 @@
 def api_sonarr_webhook(
     request: HttpRequest,
     user_profile: UserProfile,
-    payload: Dict[str, Any] = REQ(argument_type="body"),
+    payload: WildValue = REQ(argument_type="body", converter=to_wild_value),
 ) -> HttpResponse:
     body = get_body_for_http_request(payload)
     subject = get_subject_for_http_request(payload)
 
-    check_send_webhook_message(request, user_profile, subject, body, payload["eventType"])
+    check_send_webhook_message(
+        request, user_profile, subject, body, payload["eventType"].tame(check_string)
+    )
     return json_success(request)
 
 
-def get_subject_for_http_request(payload: Dict[str, Any]) -> str:
-    if payload["eventType"] != "Test" and payload["eventType"] != "Health":
-        topic = SONARR_TOPIC_TEMPLATE.format(series_title=payload["series"]["title"])
-    elif payload["eventType"] == "Test":
+def get_subject_for_http_request(payload: WildValue) -> str:
+    if (
+        payload["eventType"].tame(check_string) != "Test"
+        and payload["eventType"].tame(check_string) != "Health"
```

Comment:
For something used repeatedly like this, extract `event_type = payload["eventType"].tame(check_string)` as a variable.

Reply:
Will do!